Kernel/Devices/IDEDiskDevice.cpp

```diff
@@ -191,6 +207,10 @@ void IDEDiskDevice::initialize()
         }
     });
 
+
+    if(m_second_drive)
```

Comment:
Not sure this adds a lot of value. I guess it might have during your writing the patch, but I suppose all it does now is tell us that there's a secondary device now - it doesn't tell us which it is or anything. Maybe move it into DISK_DEBUG below, if it's still useful to keep?

Also, indentation nitpick: space after the if, before the ( -- repeats for a few other places. If you run clang-format, it should fix it all up for you.

Reply:
Yeah I thought I put that in the macro below... Seems like I forgot..